stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/channel/logic/ChannelLogic.kt

```diff
@@ -390,15 +392,16 @@ internal class ChannelLogic(
      * @param message [Message].
      */
     internal fun incrementUnreadCountIfNecessary(message: Message) {
-        val currentUserId = chatDomainImpl.user.value?.id
+        val currentUserId = GlobalMutableState.getOrCreate().user.value?.id ?: return
```

Comment:
Early return to improve readability since we don't do anything inside the function body if the user id is null.